### DISCUSSION TITLE
Error in creating a new one off campaign

### DISCUSSION THREAD

**Discussion by mkbadri** [chatwoot/chatwoot#2991] [Self hosted] [answered]
Hi team, great work on the solution. Keen to explore more and just kicked off looking at chatwoot and looks very promising

I was trying to setup some campaigns to evaluate the process but I was stuck in creating it. I am not sure if it's a bug or work in progress. Or even a configuration issue.

![image](https://user-images.githubusercontent.com/10755143/132813142-030d7d56-0309-4958-bbf0-cc30a7691a13.png)

![image](https://user-images.githubusercontent.com/10755143/132813198-b8827053-ec07-4bab-93ee-e002be7e37d5.png)

The response says "Inbox Unsupported Inbox type" while creating the campaign - what type of Inboxes are supported for campaigns - email? SMS? WhatsApp? Website?

I tried getting some info from this [documentation](https://www.chatwoot.com/docs/product/features/campaigns) but no luck on the one-off. Even in the ongoing campaign - what are the supported values?

Appreciate your assistance.

**Comment by muhsin-k** [comment, marked as answer]
@mkbadri Sorry for the trouble. There are still some improvements in the documentation are left. 
Chatwoot supports two types of campaigns now.
1. Ongoing campaigns (Supported in website live chat only.)
2. One-off campaigns (Supported in SMS inbox only)

**Reply by mkbadri** [reply]
@muhsin-k thanks for the quick revert. Noted the pending update to documentation.

For the one-off campaigns are you referring to Twilio SMS inbox only (excluding the Twilio WhatsApp Inbox)?

Meanwhile, is there a product roadmap somewhere that I can refer to - to understand when other inboxes might be supported and of course other upcoming features!

**Reply by muhsin-k** [reply]
> @muhsin-k thanks for the quick revert. Noted the pending update to documentation.
> 
> For the one-off campaigns are you referring to Twilio SMS inbox only (excluding the Twilio WhatsApp Inbox)?

Yes

> Meanwhile, is there a product roadmap somewhere that I can refer to - to understand when other inboxes might be supported and of course other upcoming features!

Please check this slides, https://docs.google.com/presentation/d/1ULRuGTax7nshuDiqXHHbKvAydsHjQS4rBLMD8yTRv_E/edit
